### PR TITLE
Add support for Ukrainian layout and edit French layout 

### PR DESCRIPTION
**Pull request by iaros**
Instead of Pull request #18 which is now messed up.

### CODE REVIEW

**Comment by iaros** [comment]
Hi, I just wanted to check the status of this pull request.

File: src/org/nyanya/android/traditionalt9/CharMap.java [hunk 1]
```diff
@@ -108,35 +116,44 @@
 
 	protected static final char[][] FRT9TABLE = {
 		{ '0', '+' },
-		{ '.', ',', '?', '!', ':', ';', '"', '/', '-', '@', '^', '€', '$', '%', '&', '*', '#',  '(', ')', '_', '1' },
-		{ 'a', 'b', 'c', 'A', 'B', 'C', 'ä', 'Ä','á', 'â', 'à', 'å', 'ç', 'Á', 'Â', 'À', 'Å', 'Ç', '2' },
```
Comment (Clam-) [inline src/org/nyanya/android/traditionalt9/CharMap.java]: Are these characters not needed to French input?

Comment (iaros) [reply src/org/nyanya/android/traditionalt9/CharMap.java]: ä, á, å do not exist in French input; æ exists in French input.
See https://fr.wikipedia.org/wiki/Alphabet_fran%C3%A7ais

Comment (Clam-) [reply src/org/nyanya/android/traditionalt9/CharMap.java]: Okay understood. Thanks for looking in to that.

**Comment by Clam-** [comment]
Thanks so much for your efforts. I assume you've been running this version on your own device without issues. I'll merge it this weekend, and update the Google Play store version too.

Thanks again for your time.

**Comment by Clam-** [comment]
Hi again,

Thanks again for this. I'm just in the process of cleaning up the gradle config and fixing some bugs I've discovered. The Play Store build may have to wait until next weekend.

Thanks again!

**Pull request merged**